legal_doc_processing/press_release/defendant/defendant/utils_defendant.py

```diff
@@ -0,0 +1,267 @@
+from transformers import pipeline
+nlp2 = pipeline("question-answering")
+
+import pandas as pd
+import numpy as np 
+import os
+import re
+import requests
+
+def _del_dummy_breaklines(txt: str):
+
+  new_txt = (
+      txt.replace("\n.", "$$$$")
+      .replace("\n", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("$$$$", "\n")
+  )
+
+  return new_txt
+
+
+def _del_bouble_breaks_and_spaces(txt: str) -> str:
+
+  new_txt = (
+      txt.replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+  )
+
+  return new_txt
+
+def _u(liste):
+  return list(set(liste))
+
+def _question_helper(txt) -> list:
+
+        _txt = txt.lower()
+        res = []
+
+        cands = [
+            "impose",
+            "judgment",
+            "order",
+            "settl",
+            "defendant",
+            "charge",
+            "against",
+            "violate",
+        ]
+
+        for cand in cands:
+            if cand in _txt:
+                res.append(cand)
+
+        return res
+
+def _question_selector(key: str) -> list:
+        """based on a key from _question helper find the list of good question to ask """
+
+        res = []
+
+        if "impose" in key:  # impose
+            res.extend(
+                [
+                    ("Who is imposed?", "who_imposed"),
+                    ("Who are imposed?", "who_imposed"),
+                    # ("what are the victims?", "what_victims"),
+                    # ("what is the victim?", "what_victim"),
```

Comment:
remove commented code for better readability

Reply:
ok je le supprime de suite

Reply:
J'ai tout juste laissé les commentaires qui expliquent le code